### PR TITLE
creted MushroomDao interface and Mushroom Entity

### PR DESCRIPTION
**Pull request by bohdancvejn**
Please check if ENUM can be part of the class Mushroom and if interface MushroomDAO needs any further changes.

### CODE REVIEW

**Review by Buvko — COMMENTED** [review]
I would suggest also adding the search by mushroom type criteria.

File: persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/Mushroom.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import java.util.Date;
+
+/**
+ * @author BohdanCvejn
+ */
+
+@Entity
+public class Mushroom {
+
+    public enum MushroomType {
```
Comment (bkompis) [inline persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/Mushroom.java]: good

File: persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/MushroomHunter.java [hunk 2]
```diff
@@ -10,15 +10,15 @@
...
-    @Column(nullable=false,unique=true)
+    @Column(nullable = false, unique = true)
     private String firstName;
 
     @NotNull
-    @Column(nullable=false,unique=true)
+    @Column(nullable = false, unique = true)
     private String surname;
 
     private boolean isAdmin;
```
Comment (bkompis) [inline persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/MushroomHunter.java]: This file should not be edited in this commit, wut whatever. Names should be not unique and not nullable.

**Review by bkompis — COMMENTED** [review]
Searching by mushroom type would be nice. MushroomHunter class should not be edited here.

**Comment by bkompis** [comment]
Please fix conflicts preventing the merge in your feature branch.

**Comment by bkompis** [comment]
Changes implemented in #44 .

**Pull request closed without merging**